docs/contributing/docs.rst

```diff
@@ -60,6 +60,12 @@ Then from the ``docs`` directory, build the HTML:
 
      $ make html
 
+If you encounter this error, it's likely that running it in virtual environment is your problem.
```

Comment:
What do you think of changing to  `it's likely that you are running inside a virtual environment` ?

Reply:
Yea, that sounds much better.